pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -196,7 +212,10 @@ def _tensornet_dtype(self):
 
         Returns: the tensor network class
         """
-        return TensorNetC128 if self.dtype == np.complex128 else TensorNetC64
+        if self.method == "tn":
+            return exactTensorNetC128 if self.dtype == np.complex128 else exactTensorNetC64
+        # self.method == "mps"
```

Comment:
Is this comment intentional? To signalize what is happening ahead, perhaps?

Reply:
I added this comment to clarify which method is used. I will change it to make it more clear